src/homepage.pug

```diff
@@ -0,0 +1,216 @@
+//-
+    This file generates a homepage for the extensions repository. 
+    It is intended to be processed by the gulp file of the repository.
+    It needs the following variables:
+    {
+        "repositoryName": "Comic Repo",
+        "repositoryDescription": "Some description here",
+        "repositoryLogo": "https://paperback.moe/icons/logo-alt.svg",
+        "baseURL": "https://yourlinkhere",
+        "sources": [{name: sourceName, tags[]: []}]
+    }
+
+    The page contains a noindex meta tag.
+
+    Can be tested with `pug -O ComicData.json -w . -o ./html -P` 
+    where ComicData.json contains information listed previously
+
+- var extensionsList = sources
+
+doctype html
+html(lang='en')
+    head
+        meta(name="robots" content="noindex")
+        meta(charset="utf-8")
+        link(rel="icon" href=repositoryLogo)
+        title= repositoryName
+    body
+        header
+            img(src=repositoryLogo, alt="Repository logo").icon
+            h1= repositoryName
+            p.pageDescription A Paperback extensions repository
+        div.content
+            p#repositoryDescription= repositoryDescription
+            blockquote
+                p A guide is available to learn everything about extensions repositories.
+                p Check it <a href="https://paperback.moe/help/guides/adding-repos/">here</a> to learn how to add and use this repository in Paperback.
+            blockquote
+                p 
+                    span On a Paperback installed device, press 
+                    //- Generate the URL allowing the app to install the repo, variables does not need to be escaped
+                    a(href="paperback://addRepo?displayName=" + repositoryName + "&url=" + baseURL) Add to Paperback
```

Comment:
Does this (line 41) end up getting escaped properly?

Reply:
Also, same line, different comment, do you think that this button is big enough? Should this be a little larger and more "Poppy" so that users eyes are drawn to it?

Reply:
I'm not sure. I was in fact able to import a repo which name contained a space. But after checking, it seems this space was not escaped in `%20`. Moreover, we could maybe have a name containing other special characters than a space. 

An idea could be to use [encodeURIComponent](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/encodeURIComponent)
```pug
a(href="paperback://addRepo?displayName=" + encodeURIComponent(repositoryName) + "&url=" + encodeURIComponent(baseURL)) Add to Paperback
```
The encoded link will be `href="paperback://addRepo?displayName=Comic%20Repo&url=https%3A%2F%2Fyourlinkhere"`

Reply:
For the second message, I totally agree! Let me see what we can do.
Maybe something like a big [Element io](https://element.eleme.io/#/en-US/component/button) or a Tachiyomi styled button.
<img width="255" alt="Capture d’écran 2020-10-22 à 10 59 26" src="https://user-images.githubusercontent.com/25133304/96849504-f596ab80-1455-11eb-9a3f-12e04023f6c6.png">

Reply:
With 6b3082c `encodeURIComponent()` is use to escape URL.
A big "Add to Paperback button" was also added in the page header in 6b3082c.

Thanks @ConradWeiser for these comments.